portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -39,13 +78,31 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     response.getWriter().print("A message from: ");
     response.getWriter().println(Arrays.toString(names));
     response.getWriter().println(Arrays.toString(words));
-  }
 
+    // Here starts the New Comment part.
```

Comment:
You now have a post request that is doing two different things, this is not good practice.
If the text response is still being used I'd suggest putting it in a different WebServlet, but it's probably not necessary anymore.